### DISCUSSION TITLE
Integrating Nextra in an Existing Next.js Project

### DISCUSSION THREAD

**Discussion by Montekkundan** [shuding/nextra#2514] [Q&A]
I have an existing project built with Next.js and I recently came across Nextra, which seems like a great framework for content-focused websites. I am interested in integrating Nextra into my existing Next.js project but couldn't find any documentation regarding this on the official Nextra guide. Is it possible to incorporate Nextra in an already set up Next.js project? If so, could anyone provide some guidance or point me to the relevant resources? Thank you in advance!

**Comment by mutasim77** [comment]
Check it out: https://nextra.site/docs/docs-theme/start#start-as-new-project

**Reply by Montekkundan** [reply]
Yes but i tried this:

```
const withNextra = require('nextra')({
  theme: 'nextra-theme-docs',
  themeConfig: './theme.config.jsx'
})
 
module.exports = withNextra()
 
// If you have other Next.js configurations, you can pass them as the parameter:
// module.exports = withNextra({ /* other next.js config */ })
```

and it did not work, it does not render out the nextconfig if i keep in inside `withNextra()`

**Reply by kriptonian1** [reply]
Did you get any solution?
